modules/FindMagnumIntegration.cmake

```diff
@@ -160,8 +160,7 @@ foreach(_component ${MagnumIntegration_FIND_COMPONENTS})
             find_package(Bullet)
             set_property(TARGET MagnumIntegration::${_component} APPEND PROPERTY
                 INTERFACE_INCLUDE_DIRECTORIES ${BULLET_INCLUDE_DIRS})
-            set_property(TARGET MagnumIntegration::${_component} APPEND PROPERTY
-                INTERFACE_LINK_LIBRARIES ${BULLET_LIBRARIES})
+            add_dependencies(MagnumIntegration::${_component} Bullet::Bullet)
```

Comment:
`add_dependencies` seems to not add the libraries for linking? @mosra Any idea? Maybe use the imported targets of the components explicity together with `INTERFACE_LINK_LIBRARIES`?